README-CONAN.md

```diff
@@ -1,108 +1,193 @@
+[env]
+```
+
+<name id="1-4"></a>
+##### 1.4) Build dependencies and install conan artefacts in your build directory</a>
+
+Execute `$ conan install` pointing to the directory containing `conanfile.py`.
 
 ```bash
-pip install conan
+$ conan install .. --build missing  # --profile msvc2017Release64
 ```
 
-For other installation methods (brew, installers, from sources), visit this [link]([install
-conan](http://docs.conan.io/en/latest/installation.html)).
+_**Visual Studio Users**_ should use `--profile msvc2017Release64`
 
-By default, conan only point to few public conan repositories. However you can add more easily. We rely in the usage of
-the **bincrafters** and **piponazo** repositories to find some of the dependencies (libcurl and XmpSdk) . Therefore, we
-will need to add it to the list of remotes
-(we only need to do this once):
+The output from this command is quite long as conan downloads or builds zlib, expat, curl and other dependencies.
+
+<name id="1-5"></a>
+##### 1.5) Execute cmake to generate build files for your environment.
 
 ```bash
-conan remote add conan-bincrafters https://api.bintray.com/conan/bincrafters/public-conan
-conan remote add conan-piponazo https://api.bintray.com/conan/piponazo/piponazo
+$ cmake ..  # -G "Visual Studio 15 2017 Win64" -DCMAKE_BUILD_TYPE=Release
 ```
 
-## Basic usage
+_**Visual Studio Users**_ should use define the Generator and Build Type:
+
+More information about CMake/Exiv2: [README-CMAKE](README-CMAKE).
+
+<name id="1-6"></a>
+##### 1.6) Build Exiv2:
 
-The first thing we need to do, is to check that conan was properly installed and you can access it:
 ```bash
-$ conan --version
-Conan version 1.1.1
+$ cmake --build . --config Release
 ```
 
-The first time you run conan, it will auto-detect your configuration and store it in a default profile in the file
-$HOME/.conan/profiles/default. In case you are using Windows, and have a installation of Visual Studio 2017, the default
-profile will look like this:
+[TOC](#TOC)
+
+<name id="2"></a>
+## 2) Platform Notes
 
-```Ini
+<name id="2-1"></a>
+### 2.1) Linux Notes
+
+##### Default Profile
+
+When you run conan install for the first time, it will detect and write the default profile ~/.conan/profile/default.  On my Ubuntu system with GCC 4.9, this is:
+
+```bash
+[settings]
+os=Linux
+os_build=Linux
+arch=x86_64
+arch_build=x86_64
+compiler=gcc
+compiler.version=4.9
+compiler.libcxx=libstdc++
+build_type=Release
+[options]
+[build_requires]
+[env]
+```
+
+##### Changing profile settings
+
+One of the most important **profile** settings to be adjusted in your conan profile when working on Linux is the field
+
+```bash
+compiler.libcxx=libstdc++11  # Possible values: libstdc++, libstdc++11, libc++
+```
+With the arrival of the c++11 standard, and the growing popularity of the *clang* compiler, it is increasingly important which version of the standard library to use (corresponds to the `-stdlib` compiler flag).
+
+Recommended **libcxx**
```

Comment:
[suggestion] Please merge these two lines. This looks weird in Markdown editors.

Reply:
I agree.  To be honest, I didn't know what that was about.  That's what it said in the original version of the document.  Thanks for pointing this out, I now understand what's being said!